File: WeatherQuadraticHardCoded.java
public class WeatherQuadraticHardCoded {
    public static void main(String[] args) {
        // Example coefficients for ax^2 + bx + c = 0
        double a = 1;
        double b = -3;
        double c = 2;
        solveQuadratic(a, b, c);
    }

    static void solveQuadratic(double a, double b, double c) {
        double discriminant = b * b - 4 * a * c;
        if (discriminant < 0) {
            System.out.println("No real roots.");
        } else {
            double root1 = (-b + Math.sqrt(discriminant)) / (2 * a);
            double root2 = (-b - Math.sqrt(discriminant)) / (2 * a);
            System.out.println("Roots: " + root1 + ", " + root2);
        }
    }
}

File: weather_quadratic_hardcoded.py
def solve_quadratic(a, b, c):
    discriminant = b ** 2 - 4 * a * c
    if discriminant < 0:
        print("No real roots.")
    else:
        root1 = (-b + discriminant**0.5) / (2 * a)
        root2 = (-b - discriminant**0.5) / (2 * a)
        print(f"Roots: {root1}, {root2}")

if __name__ == "__main__":
    # Example coefficients for ax^2 + bx + c = 0
    a = 1
    b = -3
    c = 2
    solve_quadratic(a, b, c)